zad_94/js/scripts.js

```diff
@@ -0,0 +1,57 @@
+var poziomy = 0;
+
+function drawTree(poziomy) {   
+    for (var i = 0; i < poziomy; i++) {
+        star = '';
+        for (var j = 0; j <= i; j++) {
+            star +='*';
+        }
+        console.log(star);
+    }
+}
+// wywołanie funkcji drawTree 
+drawTree(5);
+drawTree(10);
+drawTree(6);
+
+// wersja druga z pionowa choinka
+var poziomy1 = 0;
+var pion1 = '.';
+
+function drawTree1(poziomy1) {
+    if (poziomy1 === 1) {
+        poziomy1 = poziomy1 +2;
+        pion1 = '.';
+    } else if (poziomy1 === 2) {
+        poziomy1 = poziomy1 -1;
+        pion1 = '..';
+    } else if (poziomy1 === 3) {
+        //poziomy1 = poziomy1 -1;
+        pion1 = '.';
+    }
```

Comment:
Caly ten blok if/else jest zbedny, sprobuj rozwiazac zadanie za pomoca 3 petli `for`.
Kropki niech beda spacjami i mozesz wyswietlac je w petli for